Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -34,7 +34,7 @@ abstract class TypeHandling
     /**
      * @var array
      */
-    protected static $collectionTypes = ['array', \ArrayObject::class, \SplObjectStorage::class, Collection::class];
+    protected static $collectionTypes = ['array', \ArrayObject::class, \SplObjectStorage::class, Collection::class, \Traversable::class];
```

Comment:
```suggestion
    protected static $collectionTypes = ['array', \Traversable::class];
```
`ArrayObject`, `SplObjectStorage` and `Collection` already implement/extend `Traversable`

Reply:
> We could btw then even remove the \ArrayObject, \SplObjectStorage and Collection, since they all implement either \Iterator or \IteratorAggregate, but keeping them is a slight optimization for those cases.

Reply:
> We could btw then even remove the [...]

That's what I suggested :)
Or did you mean in the test case? I would keep it there to avoid future regressions.

But I think we agreed to fix this whole thing in the CollectionValidator instead, right?

Reply:
> but keeping them is a slight optimization for those cases.

But yes, let's fix it in the validator. No side-effects.